components/org.wso2.carbon.identity.oauth.ui/src/main/resources/web/oauth/edit-finish-ajaxprocessor.jsp

```diff
@@ -112,10 +115,10 @@
         }
 
     } catch (Exception e) {
-    	isError = false;
-    	String message = resourceBundle.getString("error.while.updating.app");
-    	CarbonUIMessage.sendCarbonUIMessage(message, CarbonUIMessage.ERROR, request,e);
-        forwardTo ="../admin/error.jsp";
+        isError = false;
+        String message = resourceBundle.getString("error.while.updating.app");
+        CarbonUIMessage.sendCarbonUIMessage(message, CarbonUIMessage.ERROR, request, e);
+        forwardTo = "../admin/error.jsp";
```

Comment:
can we make this a constant?

Reply:
This is a jsp page and we are redirecting to this. So no need to make this as a constant